x/leverage/keeper/liquidate_test.go

```diff
@@ -49,9 +50,9 @@ func TestComputeLiquidation(t *testing.T) {
 			tc.borrowedValue,
 		)
 
-		require.Equal(t, sdk.NewInt(expectedRepay), repay, msg+" (repay)")
-		require.Equal(t, sdk.NewInt(expectedCollateral), collateral, msg+" (collateral)")
-		require.Equal(t, sdk.NewInt(expectedReward), reward, msg+" (reward)")
+		require.True(t, sdkmath.NewInt(expectedRepay).Equal(repay), msg+" (repay)")
+		require.True(t, sdkmath.NewInt(expectedCollateral).Equal(collateral), msg+" (collateral)")
+		require.True(t, sdkmath.NewInt(expectedReward).Equal(reward), msg+" (reward)")
```

Comment:
```suggestion
		require.Equal(t, sdkmath.NewInt(expectedRepay), repay, msg+" (repay)")
		require.Equal(t, sdkmath.NewInt(expectedCollateral), collateral, msg+" (collateral)")
		require.Equal(t, sdkmath.NewInt(expectedReward), reward, msg+" (reward)")
```
This should preserve the `- expected + actual` console output on failing test, which can be helpful.

Reply:
Are you sure? `require.Equal` uses reflection and try to compare pointers (sdk.Int wraps a pointer), which are not the same.

Reply:
I've been `Equal` on `sdk.Int` and `sdk.Coin` in test suites, so unless `sdkmath.Int` will protest where `sdk.Int` worked, we should be fine.

Reply:
This works, updated